app.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const apiRouter = require("./routes/api-routers");
+const app = express();
+const { psqlErrorHandler, serverErrorHandler, customErrorHandler } = require("./error-handlers.js");
+app.use(express.json());
+app.use("/api", apiRouter);
+
+app.use((req, res, next) => {
+  res.status(404).send({ msg: 'Not Found' });
+});
+
+app.use(psqlErrorHandler);
+app.use(serverErrorHandler);
+app.use(customErrorHandler);
+
```

Comment:
Much of the code in this file is not yet needed. A good practice for developers is to only add the code needed for you ticket. Imagine someone else could take any of the other tickets, and by writing logic unnecessary for this ticket, you are actually writing some of the logic for someone else's ticket